README.md

```diff
@@ -27,70 +72,72 @@ python -m graph_notebook.static_resources.install
 python -m graph_notebook.nbextensions.install
 
 # copy premade starter notebooks
-python -m graph_notebook.notebooks.install --destination /notebook/destination/dir  
+python -m graph_notebook.notebooks.install --destination ~/notebook/destination/dir  
 
 # start jupyter
-jupyter notebook /notebook/destination/dir
+jupyter notebook ~/notebook/destination/dir
 ```
 
-## Configuration
+## Connecting to a graph database
```

Comment:
Did we want to have instructions for these steps here? Or have them moved to their own directories?

Reply:
After discussing with Brad and Karthik, we decided we wanted to have a quick "batteries included" section for 1 SPARQL endpoint, 1 Gremlin endpoint, and Neptune.  Additional setup information will still reside in the additional-databases directory.  If we could enhance the connection parameters to leave out Neptune-specific ones when not needed, it could make this section more clean.

Reply:
Let me know if that sounds ok or if you'd want to discuss further!

Reply:
Makes sense! Yeah a small section is no big deal, I wonder if it's worth embedding these into the `%graph_notebook_config` command as shortcuts? 

for instance, we could add something like:

`%graph_notebook_quickstart blazegraph` and it would set the configuration to the blazegraph setup described in this doc